src/encoder.rs

```diff
@@ -1830,10 +1833,18 @@ pub fn write_tx_blocks(fi: &FrameInvariants, fs: &mut FrameState,
 
     // TODO: these are only valid for 4:2:0
     let uv_tx_size = match bsize {
-        BlockSize::BLOCK_4X4 | BlockSize::BLOCK_8X8 => TxSize::TX_4X4,
+        BlockSize::BLOCK_4X4 | BlockSize::BLOCK_4X8 |
```

Comment:
this and the similar block below (line 1931) should/would be always the same?

Reply:
Yes, so can be done by lookup table but have different mapping for each chroma subsampling, i.e. 422 and 444 will have different. 
I expect libaom has tables for this case, sth like max_tx_size[chroma_sample_x][chroma_sample_y][block_size] --> tx_size.

Reply:
Line 702 in src/rdo.rs seems have same mapping as well.

Reply:
This issue has been resolved by #784.